src/probabilistic_estimation.py

```diff
@@ -179,41 +182,47 @@ def spcorr_structure(config):
     print('Successful structure generation. Time cost (sec):', t2-t1)
 
 
-def generate_probabilistic_estimates_serial(config, member_range=[]):
+def generate_prob_estimates_serial(config, member_range=[]):
     # most time cost comes from field_rand
 
     t1 = time.time()
 
     # parse and change configurations
-    case_name = config['case_name']
-
-    outpath_parent = config['outpath_parent']
-    path_ensemble = f'{outpath_parent}/ensemble_outputs'
+    case_name                 = config['case_name']
+    outpath_parent            = config['outpath_parent']
+    path_ensemble             = f'{outpath_parent}/ensembles/'
     os.makedirs(path_ensemble, exist_ok=True)
-    config['path_ensemble'] = path_ensemble
-    file_ens_prefix = f'{path_ensemble}/{case_name}_Ensemble_'
+    config['path_ensemble']   = path_ensemble
+    file_ens_prefix           = f'{path_ensemble}/{case_name}_ensMember_'
     config['file_ens_prefix'] = file_ens_prefix
 
-
-    path_spcorr = f'{outpath_parent}/spcorr'
+    path_spcorr                  = f'{outpath_parent}/spcorr'
     os.makedirs(path_spcorr, exist_ok=True)
-    config['path_rfweight'] = path_spcorr
-
-    file_spcorr_prefix = f'{path_spcorr}/spcorr_'
+    config['path_rfweight']      = path_spcorr
+    file_spcorr_prefix           = f'{path_spcorr}/spcorr_'
     config['file_spcorr_prefix'] = file_spcorr_prefix
 
     # in/out information to this function
-    file_stn_cc = config['file_stn_cc']
-    file_grid_reg = config['file_grid_reg']
+    file_stn_cc         = config['file_stn_cc']
+    file_grid_reg       = config['file_grid_reg']
     file_grid_auxiliary = config['file_grid_auxiliary']
-    file_ens_prefix = config['file_ens_prefix']
-    file_spcorr_prefix = file_spcorr_prefix
-
-    target_vars = config['target_vars']
+    #file_ens_prefix     = config['file_ens_prefix']   # redundant
+    #file_spcorr_prefix  = file_spcorr_prefix
+    ensemble_start      = config['ensemble_start']
+    ensemble_end        = config['ensemble_end']
+    target_vars         = config['target_vars']
     target_vars_WithProbability = config['target_vars_WithProbability']
-    ensemble_start = config['ensemble_start']
-    ensemble_end = config['ensemble_end']
-    master_seed = config['master_seed']
+    
+    if 'master_seed' in config:
+        master_seed = config['master_seed']
+    else:
+        master_seed = -1
+    
+    if 'append_date_to_output' in config:
```

Comment:
A note for myself: 
Add append_date_to_output to the configuration file and the documentation.

Reply:
And I'd like to set the default value as true so that different runs can coexist in the folder

Reply:
That's fine.  Being able to turn it off simplifies file handling when pyGMET is wrapped into other workflows, such as forecasting.